### PR TITLE
Feature/keymap/tada68

### PR DESCRIPTION
**Pull request by devinceble**
Personal Keymap for TADA68

## Description
A Personal 3 Layer TADA68 Keymap. Inspired by Tsangan and HHKB Layouts.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/tada68/keymaps/devinceble/rules.mk [hunk 1]
```diff
@@ -0,0 +1,7 @@
+MOUSEKEY_ENABLE = yes
```
Comment (fauxpark) [inline keyboards/tada68/keymaps/devinceble/rules.mk]: This file can be reduced down to just `MOUSEKEY_ENABLE = yes`, the rest are the same as the keyboard-level rules.mk.

File: keyboards/tada68/keymaps/devinceble/keymap.c [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    KC_CAPS,KC_MS_L,KC_MS_D,KC_MS_R,_______,_______,_______,_______,_______,_______,KC_LEFT,KC_RGHT,        _______,KC_END, \
+    _______,_______,_______,_______, _______,_______, _______,_______,_______,_______,KC_DOWN,_______, _______, _______, \
+    _______,_______,_______,                 _______,               _______,_______,_______,_______,_______, _______
+    ),
+
+    [2] = LAYOUT_ansi(
+    _______, KC_F1 ,KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12, KC_DEL, KC_INS ,  \
+    _______,_______, _______,_______,_______, _______,_______,_______,_______,_______,_______,_______,_______, _______,KC_HOME, \
+    _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,        _______,KC_END, \
+    _______,_______,_______,BL_DEC, BL_TOGG,BL_INC, _______,KC_VOLD,KC_VOLU,KC_MUTE,_______,_______, _______, _______, \
```
Comment (fauxpark) [inline keyboards/tada68/keymaps/devinceble/keymap.c]: The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater without them.

Comment (devinceble) [reply keyboards/tada68/keymaps/devinceble/keymap.c]: Copy Noted, I would like to keep them just to have all my personal keymaps uniformed.

File: keyboards/tada68/keymaps/devinceble/rules.mk (hunk 1 above)
Comment (devinceble) [reply keyboards/tada68/keymaps/devinceble/rules.mk]: Noted

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**